modules/Book.js

```diff
@@ -0,0 +1,117 @@
+// import {storeBooksToStorage} from './book-crud.js'
+const bookList = document.getElementById('book-list');
+const newBookForm = document.getElementById('new-book-form');
+const inputTitle = document.getElementById('input-title');
+const inputAuthor = document.getElementById('input-author');
+const successSmall = document.getElementById('success-small');
+
+class Book {
+  static booksArray = []
+
+  static count = 0
+
+  static storageUpdater;
+
+  static setUpdater(value) {
+    Book.storageUpdater = value;
+  }
+
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+
+    Book.booksArray.push(this);
+    if (typeof Book.storageUpdater === 'function') {
+      Book.storageUpdater();
+    }
+  }
+
+  deleteBook() {
+    const i = Book.booksArray.indexOf(this);
+    Book.booksArray.splice(i, 1);
+    if (typeof Book.storageUpdater === 'function') {
+      Book.storageUpdater();
+    }
+  }
+
+  static getAllBooksArray() {
+    return Book.booksArray;
+  }
+
+  static updateBooksArray(dataSource) {
```

Comment:
@sajid-munawar 
Thanks for the input. But I don't think I can name class methods with anything other than 'Static'...its to prevent them from being called by instances